python/whylogs/viz/utils/profile_viz_calculations.py

```diff
@@ -165,7 +165,7 @@ def generate_summaries(
     ref_summary: DatasetSummary = {"columns": {}, "properties": overall_stats}
     target_summary: DatasetSummary = {"columns": {}, "properties": None}
     for target_col_name in target_col_views:
-        if target_col_name in ref_col_views:
+        if target_col_name in ref_col_views and target_col_name != "image":
```

Comment:
Can you create a function that tests if a column profile view is an image metric and then do something like this?
```suggestion
        if target_col_name in ref_col_views and not (target_col_name == "image" and is_image_metric(ref_col_views[target_col_name])):
```

Reply:
I did the change before reading this, but I think it's very close to what you suggested